pymc3/stats.py

```diff
@@ -351,8 +351,8 @@ def bpic(trace, model=None):
     return 3 * mean_deviance - 2 * deviance_at_mean
 
 
-def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
-            alpha=1, seed=None, round_to=2):
+def compare(traces, models, names=None, ic='WAIC', method='stacking',
```

Comment:
Can you move that kwarg to the end? Someone might be using it like `pm.compare(traces, models, 'WAIC')`, and this would break that.
You could also check if the models have a name, and if so use those if `names` is None.